build.gradle

```diff
@@ -25,7 +25,7 @@ if (hasProperty('buildScan')) {
 }
 
 // The tools/publish-release.sh script depends on this version string being of the format "version = '1.2.3'"
-version = '0.66.0'
+version = '0.67.0'
```

Comment:
This manual version bump is so that the tests will run using the new Docker image with the NF version bump. It is a shortcoming of the testing infrastructure that it doesn't handle changes to the Dockerfile. This is tracked in [PF-877](https://broadworkbench.atlassian.net/browse/PF-877).